heron/packing/tests/java/com/twitter/heron/packing/AssertPacking.java

```diff
@@ -90,11 +90,7 @@ public static void assertNumInstances(Set<PackingPlan.ContainerPlan> containerPl
   public static void assertContainerRam(Set<PackingPlan.ContainerPlan> containerPlans,
                                         long maxRamforResources) {
     for (PackingPlan.ContainerPlan containerPlan : containerPlans) {
-      long containerRam = 0;
-      for (PackingPlan.InstancePlan instancePlan : containerPlan.getInstances()) {
-        containerRam += instancePlan.getResource().getRam();
-      }
-      Assert.assertTrue(containerRam <= maxRamforResources);
+      Assert.assertTrue(containerPlan.getRequiredResource().getRam() <= maxRamforResources);
```

Comment:
Assertions in a loop can be tricky to diagnose when they fail. Best to add a failure comment as a first arg that includes the container id, to help upon failure.